src/client/components/Avatar/Avatar.css

```diff
@@ -0,0 +1,19 @@
+.avatar-images-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+}
+.main-avatar-images {
+  max-width: 10em;
+  max-height: 10em;
+  border-bottom-left-radius: 25em;
+  border-bottom-right-radius: 25em;
+}
```

Comment:
What is the reason to use `em` here?

Reply:
Hi @bytewiz, I was adviced in a feedback given by @cecastosic to use rem or em for responsivity purposes.

Reply:
Alright, that is also fine, I would though consider letting the developer control the size of the avatar by passing props to it instead of having it fixed by the CSS. But that works too. 👍